source/adapters/level_zero/command_buffer.hpp

```diff
@@ -82,6 +82,14 @@ struct ur_exp_command_buffer_handle_t_ : public _ur_object {
   bool IsUpdatable = false;
   // Indicates if command buffer was finalized.
   bool IsFinalized = false;
+  // Command-buffer profiling is enabled.
+  bool IsProfilingEnabled = false;
+  // Command-buffer can be submitted to an in-order command-list.
+  bool IsInOrderCmdList;
```

Comment:
To be consistent with other members
```suggestion
  bool IsInOrderCmdList = false;
```